test/Streamly/Test/Prelude/Concurrent.hs

```diff
@@ -164,6 +164,10 @@ concurrentOps constr desc eq t = do
         concurrentMapM constr eq $ \n mv stream ->
             t $ S.mapM (mvarSequenceOp mv n) stream
 
+    prop1 (desc <> " sequence") $
+        concurrentMapM constr eq $ \n mv stream ->
+            t $ S.sequence $ S.map (mvarSequenceOp mv n) stream
+
```

Comment:
Can you commit this before the fix commit and ensure that the test fails?

Reply:
Not sure about the comment. Can you elaborate on this?

Reply:
ok got it. need to validate the test should fail without fix.

Reply:
Instead of failing test case hangs forever.

Reply:
After 10 min.
Prelude.Concurrent
  Stream concurrent operations
    default
      aheadly fromFoldableM
        +++ OK, passed 10 tests.
      aheadly unfoldrM
        +++ OK, passed 10 tests.
      aheadly mapM
        +++ OK, passed 10 tests.
*** Exception: thread blocked indefinitely in an MVar operation

Reply:
That's a failure, its fine.